BSP/Src/bsp.c

```diff
@@ -147,6 +168,14 @@ void CPU_CACHE_Enable(void)
   SCB_EnableDCache();
 }
 
+void DWT_Config(void)
+{
+  CoreDebug->DEMCR |= CoreDebug_DEMCR_TRCENA_Msk;
+  DWT->LAR = 0xC5ACCE55; 
```

Comment:
??? This is totally not documented anywhere, but it works most of the time. It freezes occasionally on the first us wait